modules/core/02-client/types/legacy_proposal.go

```diff
@@ -0,0 +1,60 @@
+package types
+
```

Comment:
should we add a comment to say this type is deprecated?

Reply:
Yeah I can add a block comment to this file as well!